src/styles/typography/link/index.njk

```diff
@@ -3,4 +3,4 @@ title: Links – Typography
 layout: layout-example.njk
 ---
 
-<a href="#" class="govuk-link">link text</a>
+Jump to <a href="#" class="govuk-link">HTML example</a>.
```

Comment:
The guidance says "If your link is at the end of a sentence or paragraph". I figured we didn't want a paragraph here, so added this sentence instead.

Preview:
![image](https://user-images.githubusercontent.com/72507742/170495250-507743b2-7e3d-429e-adf3-836a8927b3fc.png)
Is it ok to have 'Jump to' (or whatever we go with) outside of the link text?
Also, does 'Jump to' seem weird visually compared with the other text in the example?